samples/parallel-processing-california-housing-data/local_development/dev_dependencies.yml

```diff
@@ -0,0 +1,19 @@
+name: california_housing_dev
+dependencies:
+  - python=3.8
+  - pip=20.1.1
+  - pip:
+      - azureml-sdk
+      - azure-storage-file-datalake
+      - black
+      - flake8
+      - flake8_formatter_junit_xml
+      - isort
+      - pandas
+      - pydocstyle
+      - pytest
```

Comment:
Just a question: as you exclude testing explicitly from the example, there wouldn't be a need for pytest* dependencies, right?